### PR TITLE
Add OpenStreetMap map

### PR DESCRIPTION
**Pull request by jadtl**
This adds an OpenStreetMap map using osmdroid, the map activity displays the EPFL campus at start. Internet permission had to be added to `AndroidManifest.xml` for the map to work.

### CODE REVIEW

**Comment by jadtl** [comment]
I updated the activity to use Compose :)

File: app/src/main/java/com/github/polypoly/app/MapActivity.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package com.github.polypoly.app
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
```
Comment (Jharaxus) [inline app/src/main/java/com/github/polypoly/app/MapActivity.kt:6]: According to AndroidStudio this last import is useless and can be removed

**Review by Jharaxus — CHANGES_REQUESTED** [review]
A little detail to change but apart from that it looks good to me!

**Review by Jharaxus — APPROVED** [review]
Nice this fix the previous issue, everything looks fine now.

**Pull request merged**